modules/ROOT/pages/cloudhub-deploy-options-edit.adoc

```diff
@@ -0,0 +1,54 @@
+= CloudHub Deployment Settings
+
+If you are deploying Partner Manager applications to CloudHub, use the Partner Manager Settings to specify CloudHub deployment options for the Sandbox and Production environments, including the Mule runtime engine (Mule) version, CloudHub region, Anypoint MQ region, and so on. 
+
+== Mule Version
+
+You can specify the Mule version to use to deploy your application in the *Runtime version* drop-down. 
+
+The default and recommended option is *Use latest version*, which deploys new Partner Manager message flows using the latest Mule version. 
+
+If you want to deploy Partner Manager applications to a specific Mule version, you can select it from the drop-down list. 
+
+== CloudHub Region
+
+The default CloudHub region is retrieved from the business group’s Runtime Manager region setting in Access Manager.
+
+If you want to deploy the B2B integration flows from Partner Manager to a different region, select it from the *CloudHub Region* drop-down list before you deploy your first message flow. The CloudHub region you specify for the first deployment becomes the default setting for subsequent deployments.
+
+=== Change the Default CloudHub Region
+
+To change the default CloudHub region:
+
+. Undeploy message flows that are deployed to the currently specified CloudHub region. 
+. Redeploy the message flows after you specify a new default region.
+
+For more information about undeploying message flows, see xref:configure-message-flows.adoc[Configure Message Flows].
+
+== Anypoint MQ Region
+
+This is automatically assigned, based on the specified CloudHub region; you cannot change it.
+
+== Automatically Restart the App
+
+Select the *Automatically restart app when not responding* checkbox if you want CloudHub to automatically restart your application when the monitoring system discovers an issue.
+
+== vCore settings
+
+Configure the following vCore settings for the Sandbox and Production environments:
+
+* *Each B2B endpoint (inbound, outbound, and ack)* +
+Each unique trading partner and internal endpoint gets deployed to isolated workers, and require a minimum of *0.1 vCore* each.
+* *B2B DocumentProcessingService* +
+Core component of the Partner Manager workflow that requires a minimum of *1 vCore*.
+* *B2B APIReplicationService* +
+Core component of the Partner Manager workflow that requires a minimum of *1 vCore*.
+
+If you need assistance with sizing, contact your MuleSoft account representative.
+
+== Access Partner Manager Settings
+
+To access the deployment settings for your organization:
```

Comment:
WHY? To what purpose, or under what conditions, does your audience want or need to do this? (ditto all above)